src/content/en/updates/2017/01/payment-request-updates.md

```diff
@@ -0,0 +1,139 @@
+     }
+    }];
+
+Note: The older format will remain functional for the time being.
+
+## Chrome 56
+### pending
+As part of [payment item
+information](https://developers.google.com/web/fundamentals/getting-started/primers/payment-request/#create-paymentrequest),
+developers can add `pending` to indicate that the price is not fully determined
+yet. The field `pending` accepts a boolean value.
+
+    {
+     label: "State tax",
+     amount: { currency: "USD", value : "5.00" },
+     pending: true
+    },
+
+This is commonly used to show positions such as shipping or tax amounts that
+depend upon selection of shipping address or shipping options. Chrome indicates
+pending fields in the UI for the payment request.
+
+### requestPayerName
+As part of [shipping
+option](https://developers.google.com/web/fundamentals/getting-started/primers/payment-request/#contact-information)
+(third argument to `PaymentRequest`), developers can now add `requestPayerName`
+to request the payer's name separate from shipping address information.
+`requestPayerName` accepts a boolean value.
+
+### shippingType
+As part of [shipping
+option](https://developers.google.com/web/fundamentals/getting-started/primers/payment-request/#contact-information)
+(third argument to `PaymentRequest`), developers can now add `shippingType` to
+request the UI to show "delivery" or "pickup" instead of "shipping".
+`shippingType` accepts the strings `shipping` (default), `delivery`, or
+`pickup`.
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2017/01/payment-request-updates/delivery.png"
+    class="screenshot" alt="shippingType=&quot;delivery&quot;"/>
+  <figcaption>shippingType="delivery"</figcaption>
+</figure>
+<figure class="attempt-right">
+<img src="/web/updates/images/2017/01/payment-request-updates/pickup.png"
+  class="screenshot" alt="shippingType=&quot;pickup&quot;"/>
+  <figcaption>shippingType="pickup"</figcaption>
+</figure>
+
+### Serializer functions available to PaymentResponse and PaymentAddress
+[PaymentResponse
+object](https://developers.google.com/web/fundamentals/getting-started/primers/payment-request/#shipping-address)
+and `PaymentAddress` object are now JSON-serializable. Developers can convert
+one to a JSON object by calling the `.toJSON()` function and avoid creating
+cumbersome `toDict()` functions.
+
+    request.show().then(response =&gt; {
+     let res = response.toJSON();
+    });
+
+### canMakePayment
+In addition to the API availability, you can check to see if a user has active
+payment method before invoking Payment Request API. Remember that this is
+optional as users can still add a new payment method on Payment UI.
+
+    let request = new PaymentRequest(methods, details, options);
+    if (request.canMakePayment) {
+     request.canMakePayment().then(result =&gt; {
+       if (result) {
+         // Payment methods are available.
+       } else {
+         // Payment methods are not available.
```

Comment:
Should we change this comment to "Payment methods are not available, but users can still add a new payment method in Payment UI"?